presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -83,4 +85,18 @@ public BaseJdbcConfig setJdbcTypesMappedToVarchar(String jdbcTypesMappedToVarcha
         this.jdbcTypesMappedToVarchar = ImmutableSet.copyOf(Splitter.on(",").omitEmptyStrings().trimResults().split(nullToEmpty(jdbcTypesMappedToVarchar)));
         return this;
     }
+
+    @NotNull
+    public UnsupportedTypeHandling getUnsupportedTypeHandling()
+    {
+        return unsupportedTypeHandling;
+    }
+
+    @Config("unsupported-type-handling")
+    @ConfigDescription("Determines how unsupported type should be handled")
```

Comment:
"unsupported type" -> "an unsupported type"

or, rather, just "Unsupported type handling strategy", as in sess prop desc